packages/react-mutation-mapper/src/component/dataTable/DataTable.tsx

```diff
@@ -97,7 +97,6 @@ function getColumnVisibilityDef<T>(
     return colVisProp;
 }
 
-@observer
```

Comment:
mobx is not happy with the the `@observer` decoration added to the base DataTable. Instead we tag the components that extend `DataTable` with the decorator. Ideally we should get rid of inheritance based component implementation, but that's beyond the scope of this PR.